playbooks/system-criticality.md

```diff
@@ -31,21 +31,24 @@ Retired systems, spikes, or time-bound experiments that aren't significantly use
+| Testing           | Automated tests<br/>Tests are run on PRs and `master` by a Continuous Integration pipeline                                                                                                                 | &larr;ditto                                              | &larr;ditto +<br/>Test coverage tooling                                                                                                 |
+| Deployment        | Zero-downtime deployment                                                                                                                                                                                   | &larr;ditto +<br/>High-fidelity staging environment      | &larr;ditto +<br/>Deployment and orchestration by Kubernetes                                                                            |
+| Availability      | External availability monitoring                                                                                                                                                                           | &larr;ditto                                              | &larr;ditto                                                                                                                             |
+| Performance       |                                                                                                                                                                                                            |                                                          | Latency-based monitors and alerting (tailored to service)                                                                               |
+| Errors            |                                                                                                                                                                                                            | Error tracking                                           | &larr;ditto +<br/>Error rate alerting                                                                                                   |
+| Monitoring        |                                                                                                                                                                                                            | Application instrumentation (e.g., Datadog)              | &larr;ditto                                                                                                                             |
+| Incident handling | Standard [incident response times](/playbooks/support/on-call-slo.md) (pending improved notifications)                                                                                                     | &larr;ditto                                              | &larr;ditto +<br/>Downtime automatically reported to Opsgenie as incidents<br/>Incidents are reported and updated on public status page |
```

Comment:
We discussed adopting the same incident response-times for all levels of systems, but does that truly match our reality? In practice many [level 1](https://www.notion.so/artsy/17c4b550458a4cb8bcbf1b68060d63e6?v=cb0164f2f5e94962872d8cf7a4039732) systems aren't part of critical workflows and minor incidents may get punted to backlogs.